### PR TITLE
perturbations should be +-1 instead of 0,1

### PR DESCRIPTION
**Pull request by MSRudolph**
I think this is correct. I had it changed in most places, but not here.

### CODE REVIEW

**Review by mstechly — CHANGES_REQUESTED** [review]
Looks that we don't have any tests for gradients, would be nice to get some actually.

File: docs/examples/orqviz_tutorial_orquestra.ipynb [hunk 1]
```diff
@@ -1431,7 +1431,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```
Comment (MSRudolph) [inline docs/examples/orqviz_tutorial_orquestra.ipynb:1434]:
```suggestion
   "display_name": "Python 3",
```

File: docs/examples/orqviz_tutorial_orquestra.ipynb [hunk 2]
```diff
@@ -1445,7 +1445,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.5"
+   "version": "3.9.7"
```
Comment (MSRudolph) [inline docs/examples/orqviz_tutorial_orquestra.ipynb:1448]:
```suggestion
   "version": "3.9.5"
```

**Comment by MSRudolph** [comment]
@mstechly

File: src/orqviz/gradients.py [hunk 3]
```diff
@@ -49,15 +50,15 @@ def _gradient_function(pars: ParameterVector, direction: np.ndarray) -> float:
     grad = np.zeros_like(params)
 
     if stochastic:
-        direction = np.random.choice([0.0, 1.0], size=np.shape(params))
+        direction = np.random.choice([-1.0, 1.0], size=np.shape(params))
         grad_value = gradient_function(params, direction)
-        grad += grad_value * direction
+        grad = grad + grad_value * direction
```
Comment (mstechly) [inline src/orqviz/gradients.py:55]: Why this change?

**Review by mstechly — APPROVED** [review]

File: src/orqviz/gradients.py (hunk 3 above)
Comment (MSRudolph) [reply src/orqviz/gradients.py:55]: Manipulating in place has caused some silly problems in the past.
